composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": "^7.2|^8.0",
         "symfony/framework-bundle": "~3.4|~4.0|~5.0",
-        "doctrine/doctrine-bundle": "~1.0|~2.0",
+        "doctrine/doctrine-bundle": "^2.4.1",
```

Comment:
Is it okay to do this in a patch version? It's no BC break but will this constraint exclude many updates in projects?

Reply:
I was not sure about this, hence my comment above. Since you're not sure either, let's revert to `class_exists`